packages/app-store/metamaskweb3/lib/VideoApiAdapter.ts

```diff
@@ -0,0 +1,36 @@
+import type { VideoApiAdapterFactory } from "@calcom/types/VideoApiAdapter";
+
+/** This is a barebones factory function for a video integration */
+const ExampleVideoApiAdapter: VideoApiAdapterFactory = (credential) => {
```

Comment:
If this Video Adapter isn't being used, let's just remove it @joeauyeung

Reply:
Done, thanks for checking that out.